app/src/androidTest/java/net/aiscope/gdd_app/SampleDtoKtTest.kt

```diff
@@ -0,0 +1,28 @@
+package net.aiscope.gdd_app
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import net.aiscope.gdd_app.model.Sample
+import net.aiscope.gdd_app.network.toDto
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class SampleDtoKtTest {
+
+    @Test
+    fun shouldFillExpectedFields() {
+        val sample = Sample("_id",
+            "facility",
+            "microscopist")
+
+        //This is the DTO for sending to firebase rather than storing locally
+        val dto = sample.toDto()

```

Comment:
With the import alias this would be change to
`val dto = sample.toDtoNetwork()`